mp/game/momentum/cfg/menus/mapcmdsmenu_default.vdf

```diff
@@ -0,0 +1,49 @@
+"Map Commands Menu"
+{
+    "menu_name" "Map Commands Menu"
+    "1"
+    {
+        "label"   "#MOM_Menu_MapCommands_Toggle_Wireframes"
```

Comment:
I didn't see any localization for this?

Reply:
Crap, mustve got lost in some rebase somehow.